drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -722,16 +736,23 @@ function DragAndDropBlock(runtime, element, configuration) {
         updateDOM();
     };
 
-    var setGrabbedState = function(item_id, grabbed) {
-        for (var i = 0; i < configuration.items.length; i++) {
-            if (configuration.items[i].id === item_id) {
-                configuration.items[i].grabbed = grabbed;
+    var setGrabbedState = function(item_id, grabbed, interaction_type) {
+        for (var i = 0, item; i < configuration.items.length; i++) {
+            item = configuration.items[i];
```

Comment:
Nit: Just use `configuration.items.forEach(function(item) {` which is nicer and used elsewhere in the code.